src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly.js

```diff
@@ -12,88 +12,88 @@ import Tooltip from '../Tooltip';
 import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
 import styles from '../../styles/styles';
 import * as StyleUtils from '../../styles/StyleUtils';
-import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
-import {propTypes as anchorForCommentsOnlyPropTypes, defaultProps as anchorForCommentsOnlyDefaultProps} from './anchorForCommentsOnlyPropTypes';
+import {propTypes as anchorForCommentsOnlyPropTypes} from './anchorForCommentsOnlyPropTypes';
 import CONST from '../../CONST';
+import useWindowDimensions from '../../hooks/useWindowDimensions';
 
 const propTypes = {
     /** Press in handler for the link */
+    // eslint-disable-next-line react/require-default-props
     onPressIn: PropTypes.func,
 
     /** Press out handler for the link */
+    // eslint-disable-next-line react/require-default-props
     onPressOut: PropTypes.func,
 
     // eslint-disable-next-line react/forbid-prop-types
-    containerStyles: PropTypes.arrayOf(PropTypes.object),
-
     ...anchorForCommentsOnlyPropTypes,
-    ...windowDimensionsPropTypes,
-};
-
-const defaultProps = {
-    onPressIn: undefined,
-    onPressOut: undefined,
-    containerStyles: [],
-    ...anchorForCommentsOnlyDefaultProps,
 };
 
 /*
  * This is a default anchor component for regular links.
  */
-function BaseAnchorForCommentsOnly(props) {
+function BaseAnchorForCommentsOnly({onPressIn = undefined, onPressOut = undefined, href = '', rel = '', target = '', children = null, style = {}, onPress = undefined, ...rest}) {
```

Comment:
Contributor
 ArekChr 

Are these empty values needed as default props?

	Reply...
Contributor
 BeeMargarida Pending

Not sure if it should be undefined or empty function has default value, since that was what's normally used in defaultProps in other components

	Reply...